winrm4j/src/test/java/io/cloudsoft/winrm4j/winrm/WinRmChangePassword.java

```diff
@@ -0,0 +1,58 @@
+package io.cloudsoft.winrm4j.winrm;
+
+import com.google.common.base.Stopwatch;
+import com.google.common.collect.ImmutableList;
+import org.apache.http.client.config.AuthSchemes;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Random;
+import java.util.logging.Logger;
+
+import static org.testng.Assert.fail;
+
+public class WinRmChangePassword extends AbstractWinRmToolLiveTest {
+    private static final Logger LOG = Logger.getLogger(WinRmChangePassword.class.getName());
+
+    @Test(groups="Live")
+    public void testAuthSuccessAndFail() throws Exception {
```

Comment:
This will always fail with `NTLM` auth when reusing the context. The problem is with Brooklyn which should create a new context for the new set of credentials.